src/main/docs/guide/cloudFunction/simpleFunctions.adoc

```diff
@@ -20,4 +20,44 @@ When you extend from `FunctionInitializer` the Micronaut `ApplicationContext` wi
 
 WARNING: Functions require a no argument constructor hence you must use field injection (which requires `lateinit` in Kotlin) when injecting dependencies into the function itself.
 
-The `FunctionInitializer` super class provides numerous methods that you can override to customize how the `ApplicationContext` is built if desired.
\ No newline at end of file
+The `FunctionInitializer` super class provides numerous methods that you can override to customize how the `ApplicationContext` is built if desired.
+
+=== Running Functions Locally
+
+Raw functions cannot be executed locally. They can be tested by instantiating the function and inspecting any side effects by providing mock arguments or mocking dependent beans.
+
+=== Deployment
+
+When deploying the function to Cloud Function you should use the fully qualified name of the function class as the handler reference.
+
+First build the function with:
+
+```bash
+$ ./gradlew clean shadowJar
+```
+
+Then `cd` into the `build/libs` directory (deployment has to be done from the location where the JAR file resides):
+
+```bash
+$ cd build/libs
+```
+
+To deploy the function make sure you have `gcloud` CLI then run:
+
+```bash
+$ gcloud alpha functions deploy myfunction --entry-point example.function.Function --runtime java11 --trigger-http
```

Comment:
I think functions are now `gcloud beta`

Reply:
I'll give beta a try. Alpha works as of yesterday

Reply:
Yep beta works